tests/integration/targets/content_library/tasks/main.yaml

```diff
@@ -0,0 +1,102 @@
+---
+- set_fact:
+    datastore_name: "rw_datastore"
+    local_library_name: "local_library_001"
+
+- name: Get Datastore id required for local library creation
```

Comment:
Can you add a `_` to the task `name`, this way the task won't be injected in `vcenter_datastore_info`'s `DOCUMENTATION` block.